tmate-ssh-server.c

```diff
@@ -106,6 +107,72 @@ static ssh_channel channel_open_request_cb(ssh_session session, void *userdata)
 	return client->channel;
 }
 
+static int check_authorized_keys(const char *authorized_keys_file, struct ssh_key_struct *client_pubkey) {
+	#define MAX_PUBKEY_SIZE 0x100000
```

Comment:
It's worth mentioning this comes from: `ssh_pki_import_pubkey_file` in libssh.

This (0x100000) is the buffer size they choose to use, and they store in the heap, and use safe free after using the buffer. Choosing a different value will diverge from libssh design.

I'm fine with either.